lib/providers/vk.js

```diff
@@ -26,11 +26,15 @@ exports = module.exports = function (options) {
                 v: version
             };
 
+            if (options.fields) {
+                query.fields = options.fields;
+            }
+
             const data = await get('https://api.vk.com/method/users.get', query);
             const profile = data.response[0];
 
             credentials.profile = {
-                id: profile.uid,
+                id: profile.id,
```

Comment:
Afaik the api returns uid not id?
https://vk.com/dev/fields

Reply:
Page move from
https://vk.com/dev/fields
to
https://vk.com/dev/objects/user

```
https://vk.com/dev/users.get

Returns a list of user objects. 

A deactivated field may be returned with the value deleted or banned if a user has been suspended.
```
I think vk team move field  **uid** to field **id** in last version

Reply:
I can get to both pages by clicking through the docs so confused haha
Do you use the api? Can you validate that this change works?

Reply:
@AdriVanHoudt, yes. I use it) And this is reason why I create this pull-request)

Reply:
@AdriVanHoudt, maybe try get `id` field and if it's field not exist try get `uid` field?

Reply:
Sounds like a plan, if you change that I will merge 🎉

Reply:
@AdriVanHoudt, what the problem with travis tests?